renv.lock

```diff
@@ -659,16 +631,15 @@
     },
     "projectlive.modules": {
       "Package": "projectlive.modules",
-      "Version": "0.7.4",
+      "Version": "0.10.1",
```

Comment:
When I deploy this branch locally and select "Ray Mattingly" 's study I see this:
<img width="1736" alt="Screen Shot 2022-04-07 at 5 38 07 PM" src="https://user-images.githubusercontent.com/17917521/162341579-5ea96e99-d127-4959-bea7-da4bb1076e81.png">
<img width="1689" alt="Screen Shot 2022-04-07 at 5 38 24 PM" src="https://user-images.githubusercontent.com/17917521/162341586-850fe535-20ec-40b0-8b6f-1671e61a1a4a.png">

The sections in this module are not populating for any of the studies.
I have not gotten any error on my end which can explain why this might be.

At the same time seeing some other errors like : 
On Snapshot page:

<img width="1679" alt="Screen Shot 2022-04-07 at 5 49 31 PM" src="https://user-images.githubusercontent.com/17917521/162342056-bb81e9e1-b053-455b-b0bc-d2a0f02097c2.png">

on new submissions page:

<img width="1684" alt="Screen Shot 2022-04-07 at 5 50 23 PM" src="https://user-images.githubusercontent.com/17917521/162342059-86ec41bc-8f9b-4d35-8794-a060c53499fd.png">